examples/producer/src/main/java/io/strimzi/examples/producer/ExampleProducer.java

```diff
@@ -114,11 +114,17 @@ private static Properties buildProducerConfig() {
 
         Properties p = new Properties();
 
-        p.setProperty("security.protocol", "SASL_PLAINTEXT");
-        p.setProperty("sasl.mechanism", "OAUTHBEARER");
-        p.setProperty("sasl.jaas.config", "org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required ;");
-        p.setProperty("sasl.login.callback.handler.class", "io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler");
+        String username = "myclient";
+        String password = "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJ2blhyakpIelBaQU04cVhoTXF4Rm4tcGRoSUptSFZVTFd2NjlKQW52M1Q0In0.eyJleHAiOjE2MDUyODUxNzgsImlhdCI6MTYwNDQyMTE3OCwianRpIjoiOWI5MjdlYzgtMDBiMi00YmVkLWI5OGMtNDYwYTc0Y2I0OGE5IiwiaXNzIjoiaHR0cDovL2tleWNsb2FrOjgwODAvYXV0aC9yZWFsbXMva2Fma2EtYXV0aHoiLCJhdWQiOiJhY2NvdW50Iiwic3ViIjoiZTU3MTY5NjUtNDljYi00NjhjLWJhYmQtOWI0ODcwYWY1NjVmIiwidHlwIjoiQmVhcmVyIiwiYXpwIjoibXljbGllbnQiLCJzZXNzaW9uX3N0YXRlIjoiZThmYmZmN2ItYTUwMy00ZDRkLWIwOTktNmVhYzU0ZTdmZTlmIiwiYWNyIjoiMSIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJvZmZsaW5lX2FjY2VzcyJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoicHJvZmlsZSBlbWFpbCIsImVtYWlsX3ZlcmlmaWVkIjpmYWxzZSwiY2xpZW50SWQiOiJteWNsaWVudCIsImNsaWVudEhvc3QiOiIxNzIuMTguMC4yIiwicHJlZmVycmVkX3VzZXJuYW1lIjoic2VydmljZS1hY2NvdW50LW15Y2xpZW50IiwiY2xpZW50QWRkcmVzcyI6IjE3Mi4xOC4wLjIifQ.UaCVWWRAe9r0LNJfyW3M5C95fs_hlJ2CmgbbeKBRw5vnTPIWz294JJCQjeuXlABUkn2JraoETN6226rBFFaudZFNzmx2BAYziTvvo18lV9oKFf2ZPVKWDaiM0sih1sDBlamfFTNtOP2CPMNubqSKCMP-_dH5ffLpE9d38atQ8bCTKF9oSQZzRELDwftg5jnQssLaB9hgiVcbslH1eEm2uTY9ps2uBNE7Lq8_NOERSaqIy09haYqeRAZwtWvzs7L6pmiWAThlhdYICMUSYoe-7cOkLEZdRTyLdpIuso_tB1qDt8cem7vev8y_zSVi7DFqFtQv2R4VSKVAx3aFdNbUXQ";
+
+        //p.setProperty("security.protocol", "SASL_PLAINTEXT");
+        //p.setProperty("sasl.mechanism", "OAUTHBEARER");
+        //p.setProperty("sasl.jaas.config", "org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required ;");
+        //p.setProperty("sasl.login.callback.handler.class", "io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler");
```

Comment:
Is it intentional to have single example with both OAUTHBEARER and PLAIN in one like this? If yes, it might be nice to add some explanatory comments here in the code to explain what and when should be un-commented and what should be commented.

Reply:
Good catch, this was not intended this way.